source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Config.daml

```diff
@@ -26,12 +26,12 @@
         <!-- add your controls here -->
         <button id="ProAppDistanceAndDirectionModule_DistanceAndDirectionDockpane_ShowButton" caption="Distance and Direction" className="DistanceAndDirectionDockpane_ShowButton" loadOnClick="true" smallImage="Images\distance-direction-16.png" largeImage="Images\distance-direction-32.png">
         </button>
-        <button id="ProAppDistanceAndDirectionModule_PropertiesButton" caption="Properties" className="PropertiesButton" loadOnClick="true" smallImage="Images\GenericButtonBlue16.png" largeImage="Images\GenericButtonBlue32.png">
+        <!--<button id="ProAppDistanceAndDirectionModule_PropertiesButton" caption="Properties" className="PropertiesButton" loadOnClick="true" smallImage="Images\GenericButtonBlue16.png" largeImage="Images\GenericButtonBlue32.png">
```

Comment:
If these are being removed permanently, you may just want to delete the lines, rather than comment out. If keeping, then probably just add a comment line like: "uncomment these lines if needed for debugging of ..."